novuhq/website
# Pricing page update

This pull request brings some small updates for pricing page:

1. Remove the `Cloud / Self Hosted` tubmler in hero
2. Add “Translation” under “Advanced Features” in pricing table

Steps to check:

- open [preivew](https://deploy-preview-137--novu-website.netlify.app/pricing/)
- make sure that everything looks and works as expected
